### ISSUE TITLE
<?php ?> recognized as expression inside <script> tags

### ISSUE THREAD

**Issue by odahcam** [atom/language-php#152]
The `PHP` tags are recognized as `JS` expressions/keywords inside `<script>` tags, as you can see:
![image](https://cloud.githubusercontent.com/assets/3942006/18012462/f057f53e-6b8f-11e6-8a9c-9e95834035e1.png)
![image](https://cloud.githubusercontent.com/assets/3942006/18012553/6a931b44-6b90-11e6-84d2-ec610c52c64b.png)

Only if the `<?php` tag is closed in the same line this happens, instead, if you close the tag in a different line it works fine. See the examples bellow:
![image](https://cloud.githubusercontent.com/assets/3942006/18012632/c9b5b6fe-6b90-11e6-9ab2-c58e0514bc3a.png)

The expected markdown for a `PHP` tag: (line 105 of above example)
![image](https://cloud.githubusercontent.com/assets/3942006/18012687/056ec758-6b91-11e6-8582-be8b08830cc2.png)

The unexpected behavior: (line 103)
![image](https://cloud.githubusercontent.com/assets/3942006/18012742/2868876c-6b91-11e6-9c03-d4c7619610fa.png)

Used snippet:

``` html
    <script>
    
        <?php $some_php_var ?> // closed same line

        <?php
            $some_php_var
        ?> // closed different line
            
        <?php $some_php_var
        ?> // closed different line

        <?php $some_php_var

        <?php $some_php_var ?> // closed different line

        <?php $some_php_var  // not closed

    </script>

**Comment by Ingramz** [comment]
@50Wliu this seems to be fixed.

**Comment by 50Wliu** [comment]
:sparkles:

**Comment by odahcam** [comment]
Can't see the fix:
![image](https://cloud.githubusercontent.com/assets/3942006/21767168/9692a910-d659-11e6-8cbd-3c1ac9f8b250.png)

Atom Version 1.12.9

**Comment by Ben3eeE** [comment]
@odahcam I believe the fix is in Atom 1.13 which is currently in beta.

**Comment by odahcam** [comment]
For sure this is fixed, thanks guys.

![image](https://cloud.githubusercontent.com/assets/3942006/21984229/1dec4f4c-dbdc-11e6-9940-547939f84eaa.png)

----

My theme: https://atom.io/themes/atom-sublime-monokai-syntax
